pkgs/development/python-modules/fcmaes/default.nix

```diff
@@ -0,0 +1,61 @@
+{ lib
+, armadillo
+, buildPythonPackage
+, cmake
+, fetchPypi
+, mkl
+, pytestCheckHook
+, scipy
+}:
+
+buildPythonPackage rec {
+  pname = "fcmaes";
+  version = "0.9.5.7";
```

Comment:
```suggestion
  version = "0.9.5.7";
  disabled = isPy27;
```

```
builder for '/nix/store/2rcynq1n9qzpmrdikcpcqnsd3jsw2bxl-python2.7-fcmaes-0.9.5.7.drv' failed with exit code 1; last 10 log lines:
                                      ^
  SyntaxError: invalid syntax


  ----------------------------------------------------------------------
  Ran 1 test in 0.000s

  FAILED (errors=1)
```

Reply:
isPy27 added

for python38, I tried `nix-build -A python38Packages.fcmaes` and `nixpkgs-review pr 83787` but I didn't get the error